db/migrate/20180614232518_drop_fields.rb

```diff
@@ -0,0 +1,5 @@
+class DropFields < ActiveRecord::Migration[5.1]
+  def change
+    drop_table :fields
```

Comment:
Does this have to be dropped in order to make the joins table?

Reply:
See the following migration. I re-added the fields table, but without ID.

Reply:
Nvm I see you drop it to recreate with `id: false`.